services/lsst-dev.rst

```diff
@@ -209,6 +210,22 @@ Note that the SSD and GPFS-backed stacks are independent: while both will automa
 
 Administrators may wish to note that the shared stack is automatically updated using the script :file:`~lsstsw/shared-stack/shared_stack.py`, which is executed nightly by Cron.
 
+.. _lsst-dev-gitlfs:
+
+Configure Git LFS
+=================
+
+You can get Git LFS through EUPS:
+
+.. code-block:: bash
+
+   setup -j git_lfs
```

Comment:
I don't think you actually need the `-j` here (although it shouldn't do any harm).